projects/packages/jetpack-mu-wpcom/src/features/launch-button/index.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Adds a "launch site" button to the admin bar.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+use Automattic\Jetpack\Status;
+
+/**
+ * Adds a "launch site" button to the admin bar.
+ *
+ * @param WP_Admin_Bar $admin_bar The WordPress admin bar.
+ */
+function wpcom_add_launch_button_to_admin_bar( WP_Admin_Bar $admin_bar ) {
+	if ( ! current_user_can( 'manage_options' ) ) {
+		return;
+	}
+	$is_launched = get_option( 'launch-status' ) !== 'unlaunched';
+	if ( defined( 'IS_ATOMIC' ) && IS_ATOMIC ) {
+		$blog_domain = ( new Status() )->get_site_suffix();
+	} else {
+		$blog_id     = get_current_blog_id();
+		$blog_domain = preg_replace( '!^https?://!', '', get_primary_redirect( $blog_id ) );
```

Comment:
You can use `wp_parse_url( home_url(), PHP_URL_HOST )` I think?
This is what the site management widget currently uses (untouched by me).
Also I think it's using that for both simple and atomic.

Reply:
I copied this code from the previous launch bar we had in the frontend but I can try your suggestion. There's so many ways of doing the same things and no reference for these things. I think we need this reference at some point.